kimchi/src/circuits/expr.rs

```diff
@@ -2343,7 +2343,7 @@ where
             JointCombiner => "joint_combiner".to_string(),
             EndoCoefficient => "endo_coefficient".to_string(),
             Mds { row, col } => format!("mds({row}, {col})"),
-            Literal(x) => format!("field(\"0x{}\")", x.into_repr()),
+            Literal(x) => format!("field(\"0x{}\")", x.into_bigint()),
```

Comment:
yeah looks like that's the change that's breaking things, I guess we need a to_hex function in `field_helpers.rs`

Reply:
oh looks like we already have one : o